action.yml

```diff
@@ -14,14 +14,17 @@ inputs:
   target-dir:
     description: "The target dir that should be cleaned and persisted, defaults to `./target`"
     required: false
+  cache-on-failure:
```

Comment:
can you put this into the readme as well?

Reply:
Done